src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -108,6 +106,10 @@ export default class ProcessConverter extends AbstractConverter<Process> {
     });
   }
 
+  private isEvent(kind: ShapeBpmnElementKind): boolean {
+    return [ShapeBpmnElementKind.EVENT_START, ShapeBpmnElementKind.EVENT_END].includes(kind);
```

Comment:
:question: shouldn't we put the array in a constant? I don't know the performance/load impacts of recreating arrays everytime vs keep a ref to a constant array

Reply:
:heavy_check_mark: as per team decision, keep it as it is for now